### ISSUE TITLE
OrbitDB Stickers

### ISSUE THREAD

**Issue by RichardLitt** [orbitdb/orbitdb#577]
**Announcment!** We have stickers based on [our logo](https://github.com/orbitdb/logo). 

![img_2405_1024](https://user-images.githubusercontent.com/910753/53825547-8b5d8a00-3f44-11e9-885f-63997de746b7.jpg)

I'd be more than happy to send anyone some stickers, for free. Just let me know how many you want - 100, or 1000? - and I can have some printed and mailed to you. 

Send me your address via email to richardlitt at orbitdb.org. Sound good? 

These also don't have to be hexagons, although that's our preferred style. If you want a circular sticker, lmk.

**Comment by haadcode** [comment]
Closing as obsolete.

**Comment by RichardLitt** [comment]
Actually, I still have a lot of these. I mailed some a couple of months ago to someone who wanted one.

**Comment by nicocossiom** [comment]
> Actually, I still have a lot of these. I mailed some a couple of months ago to someone who wanted one.

@RichardLitt I would appreciate if you could send me one one or two if possible. For my contact info you can reach me at nicocossiom@gmail.com, ty in advanced!

**Comment by Ann-Teresa** [comment]
Hi Richard, I sent you a private message as I would like to have an amount of stickers to eventually hand out.  Thank you :)

**Comment by RichardLitt** [comment]
Thanks @Ann-Teresa. I'm not seeing that PM - where did you send it?

**Comment by RichardLitt** [comment]
Note: I updated me email address. The old orbitdb.org one is dead.

**Comment by Ann-Teresa** [comment]
Hi @RichardLitt , I sent it on matrix but no worries I will send you an email.  Thank you :)

**Comment by riskpumpfun-debug** [comment]
Yes sir Sent from my iPhoneOn 17 Feb 2026, at 04:33, Richard Littauer ***@***.***> wrote:﻿RichardLitt left a comment (orbitdb/orbitdb#577)
:D Are there new stickers?
